ql/src/test/queries/clientpositive/materialized_view_create_rewrite_5.q

```diff
@@ -11,6 +11,8 @@ set hive.materializedview.rewriting=true;
 create table cmv_basetable_n6 (a int, b varchar(256), c decimal(10,2), d int) stored as orc TBLPROPERTIES ('transactional'='true');
 
 insert into cmv_basetable_n6 values
+ (1, 'alfred', 10.30, 2),
+ (1, 'alfred', 10.30, 2),
```

Comment:
Instead of modifying existing tests to add duplicates in order to trigger the problem I think it would be better to add a new minimal test like the one you posted last under the JIRA ticket.

Reply:
I added the 2nd example from the jira as a new q test but I also kept the change of this one because we ususally face extra challenges in case of MVs with Join.